.github/workflows/pull-request-checks.yaml

```diff
@@ -78,7 +78,7 @@ jobs:
           DEBIAN_FRONTEND: noninteractive
         run: |
           sudo apt-get update
-          sudo apt-get install --no-install-recommends -yq clang-10 clang++-10 gdb jq flex bison libxml2-utils cpanminus ccache z3
+          sudo apt-get install --no-install-recommends -yq clang-19 clang++-19 gdb jq flex bison libxml2-utils cpanminus ccache z3
```

Comment:
Perhaps this should now just be `clang` and `clang++` instead of using a particular version of the compiler?

Reply:
That was my first attempt -- `apt-get install` errors.